docs/tutorials/Geospatial with Fused/gee_bigquery.mdx

```diff
@@ -144,4 +144,44 @@ def udf(bounds: fused.types.TileGDF=None, geography_column=None):
         return client.query(query).to_geodataframe(geography_column=geography_column)
     else:
         return client.query(query).to_dataframe()
+```
+
+
+## Big Query - Option 2: Secrets
+
+If you're GCP Project already has the correct permissions, you can use the [`fused.secrets`](/workbench/preferences/#secrets-management) object to access your GCP secrets.
```

Comment:
```suggestion
If your GCP Project already has the correct permissions, you can use the [`fused.secrets`](/workbench/preferences/#secrets-management) object to access your GCP secrets.
```
This seems to be confusing the GCP IAM permissions and whether the service account is loaded into fused.secrets.